api/handlers/service_binding_test.go

```diff
@@ -397,6 +397,50 @@ var _ = Describe("ServiceBinding", func() {
 		})
 	})
 
+	Describe("GET /v3/service_credential_bindings/{guid}/parameters", func() {
+		BeforeEach(func() {
+			requestMethod = http.MethodGet
+			requestBody = ""
+			requestPath = "/v3/service_credential_bindings/service-binding-guid/parameters"
+
+			serviceBindingRepo.GetServiceBindingParametersReturns(repositories.ServiceBindingParametersRecord{
+				Parameters: map[string]any{
+					"foo": "val1",
+					"bar": "val2",
+				},
+			}, nil)
+		})
+
+		It("returns the service binding parameters", func() {
+			Expect(rr).To(HaveHTTPStatus(http.StatusOK))
+			Expect(rr).To(HaveHTTPHeaderWithValue("Content-Type", "application/json"))
+			Expect(rr).To(HaveHTTPBody(SatisfyAll(
+				MatchJSONPath("$.parameters.foo", "val1"),
```

Comment:
According to the [docs](https://v3-apidocs.cloudfoundry.org/version/3.159.0/index.html#get-parameters-for-a-service-credential-binding) there is no `parameters` key in the json reponse of `/v3/service_credential_bindings/[guid]/parameters`. Instead the params json is presented as-is to the user. The cloud controller implementation does not even define a presenter for this particular endpoint.